hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -21,13 +21,17 @@ To boot from NVMe you need pre-release versions of the bootloader, VideoCore fir
 
 ### Bootloader
 
-You need to use `rpiboot` to update the CM4 bootloader. Instructions for building `rpiboot` and configuring the IO board to switch the ROM to usbboot mode are in the [usbboot Github repository](https://github.com/raspberrypi/usbboot).
+You need to use `rpiboot` to update the CM4 bootloader to use the beta bootloader. If you are using a CM4 with an eMMC then you must also change the BOOT_ORDER to give NVMe priority, otherwise the CM4 will continue to boot from eMMC.
```

Comment:
I think it's worth mentioning _here_ that rpiboot must be run on another PC here, perhaps removing some of us duplication from the following paragraphs.
E.g. "Another computer running rpiboot must be used to update the EEPROM on a CM4".